.buildkite/scripts/steps/code_coverage/ingest.sh

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+source .buildkite/scripts/common/util.sh
+
+is_test_execution_step
```

Comment:
You probably don't need this for this script. It's used for any steps that run tests, to make sure certain tasks are run, like running the failed test reporter and uploading test artifacts.

Reply:
I actually upload merged reports as test artifacts (lines 50-51)
```
tar -czf target/kibana-coverage/jest/kibana-jest-coverage.tar.gz target/kibana-coverage/jest-combined
tar -czf target/kibana-coverage/functional/kibana-functional-coverage.tar.gz target/kibana-coverage/functional-combined
```

Reply:
I would probably just add `buildkite-agent artifact upload 'target/kibana-coverage/functional/**/*'` (or whatever you need) after the files get generated, since I don't think you need/want any of the other `is_test_execution_step` stuff. It's up to you, either way works for me